src/test/java/uk/gov/companieshouse/api/accounts/controller/IntangibleAssetsControllerTest.java

```diff
@@ -0,0 +1,360 @@
+import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
+import uk.gov.companieshouse.api.accounts.utility.ErrorMapper;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class IntangibleAssetsControllerTest {
+
+    @Mock
+    private IntangibleAssetsService intangibleAssetsService;
+
+    @Mock
+    private ApiResponseMapper apiResponseMapper;
+
+    @Mock
+    private ErrorMapper errorMapper;
+
+    @Mock
+    private IntangibleAssets intangibleAssets;
+
+    @Mock
+    private BindingResult bindingResult;
+
+    @Mock
+    private HttpServletRequest request;
+
+    @Mock
+    private Transaction transaction;
+
+    @Mock
+    private Errors errors;
+
+    @Mock
+    private SmallFull smallFull;
+
+    @Mock
+    private Map<String, String> links;
+
+    @InjectMocks
+    private IntangibleAssetsController controller;
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+    private static final String INTANGIBLE_ASSETS_LINK = "intangibleAssetsLink";
+
+    @Test
+    @DisplayName("Tests the successful creation of intangible assets resource")
+    void createIntangibleAssetsSuccess() throws DataException {
+        when(bindingResult.hasErrors()).thenReturn(false);
+        when(request.getAttribute(AttributeName.TRANSACTION.getValue())).thenReturn(transaction);
+
+        ResponseObject responseObject = new ResponseObject(ResponseStatus.CREATED, intangibleAssets);
+        when(intangibleAssetsService.create(intangibleAssets, transaction, COMPANY_ACCOUNTS_ID, request))
+                .thenReturn(responseObject);
+
+        ResponseEntity responseEntity = ResponseEntity.status(HttpStatus.CREATED).body(responseObject.getData());
+        when(apiResponseMapper.map(responseObject.getStatus(), responseObject.getData(), responseObject.getErrors()))
+                .thenReturn(responseEntity);
+
+        ResponseEntity response =
+                controller.create(intangibleAssets, bindingResult, COMPANY_ACCOUNTS_ID, request);
+
+        assertNotNull(response);
+        assertEquals(HttpStatus.CREATED, response.getStatusCode());
+        assertEquals(intangibleAssets, response.getBody());
+
+        verify(errorMapper, never()).mapBindingResultErrorsToErrorModel(bindingResult);
+        verify(intangibleAssetsService, times(1))
```

Comment:
You shouldn't need to verify this behaviour since you've asserted the method has been executed in your `when / then` statement. If `intangibleAssetsService#create` did not execute, junit jupiter will throw an exception before this line is hit, making it defunct